deploy_and_run_tests.sh

```diff
@@ -227,6 +227,11 @@ if [[ -n "${run_unit_tests}" ]]; then
 fi
 pip install --upgrade .[int_test]
 
+# Due to the dependency issue, we have to manually upgrade httplib2 in order to
+# circumvent the CERTIFICATE_VERIFY_FAILED error. For more info, visit
+# https://github.com/googlegenomics/gcp-variant-transforms/issues/453.
+pip install --upgrade httplib2
```

Comment:
# Force an upgrade to avoid SSL certificate verification errors (issue #453).

I think that will (a) be sufficient and (b) autolink to the issue?

Reply:
Done.